sphinx/source/docs/user_guide/server.rst

```diff
@@ -882,92 +860,91 @@ tunnel to the remote host:
+be able to navigate to ``localhost:5006`` as though the Bokeh server were
+running on the local machine.
+
+A slightly more complicated scenario involves a gateway between the server and
+the local machine. In that situation, a reverse tunnel must be established from
+the server to the gateway with another tunnel connecting the gateway with the
+local machine.
 
 Issue the following commands on the **remote host** where the Bokeh server
-will run:
+will be running:
 
 .. code-block:: sh
 
     nohup bokeh server &
     ssh -NfR 5006:localhost:5006 user@gateway.host
 
 Replace *user* with your username on the gateway and *gateway.host* with the
-hostname/IP address of the gateway. You may be prompted for login credentials
-for the gateway.
+hostname or IP address of the gateway. The gateway may prompt you for login
+credentials.
 
-Now set up the other half of the tunnel, from the local machine to the
-gateway. On the **local machine**:
+To setup the tunnel between the local machine and the gateway, run the
+following command on the **local machine**:
 
 .. code-block:: sh
 
     ssh -NfL localhost:5006:localhost:5006 user@gateway.host
 
 Again, replace *user* with your username on the gateway and *gateway.host*
-with the hostname/IP address of the gateway. You should now be able to access
-the Bokeh server from the local machine as if the Bokeh server were running
-on the local machine by navigating to ``localhost:5006`` on the local machine.
-You can even set up client connections from a Jupyter notebook running on the
-local machine.
+with the hostname or IP address of the gateway.
+
+You should now be able to access the Bokeh server from the local machine by
+navigating to ``localhost:5006``. You can even set up client connections from
+a Jupyter notebook running on the local machine.
 
 .. note::
     We intend to expand this section with more guidance for other tools and
     configurations. If you have experience with other web deployment scenarios
-    and wish to contribute your knowledge here, please
-    contact us on https://discourse.bokeh.org
+    and wish to contribute your knowledge here, please contact us on
+    https://discourse.bokeh.org
 
 .. _userguide_server_deployment_ssl:
 
-SSL Termination
+SSL termination
 ~~~~~~~~~~~~~~~
 
-A Bokeh server can be configured to terminate SSL connections (i.e. to service
-secure HTTPS and WSS sessions) directly. At a minimum, the ``--ssl-certfile``
-argument must be supplied. The value must be the path to a single file in PEM
-format containing the certificate as well as any number of CA certificates
-needed to establish the certificate's authenticity:
+You can configure the Bokeh server to terminate SSL connections and serve
+secure HTTPS and WSS sessions directly. To do so, you'll have to supply the
+``--ssl-certfile`` argument with the value of the path to a single PEM file
+containing a certificate as well as any number of CA certificates needed to
```

Comment:
Might be nice for "CA certificates" to link to https://en.wikipedia.org/wiki/Certificate_authority or similar